<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1163</width>
    <height>879</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="1" column="1">
             <widget class="QLineEdit" name="x_lineedit"/>
            </item>
            <item row="5" column="1">
             <widget class="QLineEdit" name="z_lineedit"/>
            </item>
            <item row="2" column="2" rowspan="2">
             <widget class="QPushButton" name="y_push_button">
              <property name="text">
               <string>Brows</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="y_lineedit"/>
            </item>
            <item row="0" column="2" rowspan="2">
             <widget class="QPushButton" name="x_push_button">
              <property name="text">
               <string>Brows</string>
              </property>
             </widget>
            </item>
            <item row="4" column="2" rowspan="2">
             <widget class="QPushButton" name="z_push_button">
              <property name="text">
               <string>Brows</string>
              </property>
             </widget>
            </item>
            <item row="6" column="1" colspan="2">
             <widget class="QDialogButtonBox" name="buttonBox">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="standardButtons">
               <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
              </property>
             </widget>
            </item>
            <item row="7" column="1" colspan="2">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>228</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="0" rowspan="2">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>x</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0" rowspan="2">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>y</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0" rowspan="2">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>z</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTableWidget" name="earthquake_prop_table_widget">
            <row>
             <property name="text">
              <string>name</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>date</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>station</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>No. points</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>dt</string>
             </property>
            </row>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget_2">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="PlotWidget" name="x_accelerated"/>
          </item>
          <item>
           <widget class="PlotWidget" name="y_accelerated"/>
          </item>
          <item>
           <widget class="PlotWidget" name="z_accelerated"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
